src/OrchardCore/OrchardCore.Notifications.Core/Extensions/CacheTagExtensions.cs

```diff
@@ -0,0 +1,9 @@
+using OrchardCore.Notifications;
+
+namespace OrchardCore.Environment.Cache;
+
+public static class CacheTagExtensions
+{
+    public static Task RemoveUnreadUserNotificationTagAsync(this ITagCache tagCache, string username)
```

Comment:
I think a static helper method returning the name of the name might be more useful since it could be used for adding and removing. There is not much value otherwise in hiding that `RemoveAsync` call. This can be done directly on the service from the callsite.

Reply:
done.